package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "laravel-mix-pug",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
This line is same state as master branch.
The target branch of this pull request is develop.
Please merge master branch to develop branch, before accepting this pull request.